examples/DotNetCore/AzureFunction/FunctionApp/Function1.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionApp
+{
+    public static class Function1
+    {
+        static readonly MyConfig _myConfig = new MyConfig();
```

Comment:
>new MyConfig(); [](start = 45, length = 15)

If we do this can we just add 

```
        public IConfiguration Configuration { set; get; }
        public IConfigurationRefresher ConfigurationRefresher { set; get; }
```

as static members and then use a static constructor to initialize them.

Reply:
Then we can avoid the extra type.

---
In reply to: [346051128](https://github.com/Azure/AppConfiguration/pull/187#discussion_r346051128) [](ancestors = 346051128)

Reply:
Good idea.